tests/jballerina-unit-test/src/test/resources/test-src/expressions/binaryoperations/integer_range_operators.bal

```diff
@@ -488,6 +488,73 @@ function testIsolatednessOfRangeExprIterableAndIterator() {
     } _ = 1 ..< 2;
 }
 
+
```

Comment:
Let's add tests for when the range function is used outside an iterable context also. E.g., iterable variable declaration and usage